custom.d.ts

```diff
@@ -1,7 +1,5 @@
 ///<reference types="chrome"/>
 
-type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
This was added in TS 3.5: https://www.infoq.com/news/2019/06/typescript-3-5-release/